packages/terra-clinical-item-view/tests/wdio/item-view-spec.js

```diff
@@ -0,0 +1,97 @@
+/* global before, browser, Terra */
+
+const viewports = Terra.viewports('tiny', 'small', 'medium', 'large', 'huge', 'enormous');
+// Issue occurring because of common accessoryEnd in the test examples.
+const rules = { 'duplicate-id': { enabled: false } };
```

Comment:
Should we just refactor the id's to be unique in the example test files, instead of turning this rule off?

Reply:
Once this rule is removed, I think everything looks good.

Reply:
I'm not sure if it's the best thing to do. Currently, we use common `startAccessory` and `endAccessory` . We'll end up creating multiple ones of them for no reason. Moreover, the impact of this is just moderate. 
https://github.com/cerner/terra-clinical/blob/master/packages/terra-clinical-item-view/examples/test-examples/OverflowDisplaysItemView.example.jsx#L14

Any thoughts?